front-end-tools/src/reading_from_blockchain.ts

```diff
@@ -252,8 +252,8 @@ export async function read(
     }
 
     if (moduleSchema === undefined) {
-        // If no schema is provided return the raw bytes
-        return JSONbig.stringify(res.returnValue);
+        // If no schema is provided return the raw bytes as a hex string.
+        return JSONbig.stringify(`0x${uint8ArrayToHexString(res.returnValue.buffer)}`);
```

Comment:
So it will return a double-quoted string? Why double quotes are needed?

Reply:
good point, a pure `a double-quoted string` doesn't look good (and there is no information on what this string represents).

I return an object now that has a human-readable field name `rawBytes` instead as follows:
![Screenshot from 2024-06-13 11-28-28](https://github.com/Concordium/concordium-smart-contract-tools/assets/13039169/f5407592-4038-4973-b7f1-625d0de4b748)